spread.yaml

```diff
@@ -9,7 +9,6 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
```

Comment:
But is there a clear benefit in not being explicit here, given we have a few tests explicitly checking that the reexec functionality works properly? It feels somewhat easy to do a major mistake and not test the snapd just built at all, and release something absurdly broken.